### PR TITLE
Login before getting the list of channels

### PR DESCRIPTION
**Pull request by cbosdo**
The proxy assumes the client first calls login to cache the
authentication token headers... we'ld better call login before or the
listChannels would fail.

Related issues:
 * https://bugzilla.suse.com/show_bug.cgi?id=1193585
 * https://bugzilla.suse.com/show_bug.cgi?id=1197963

### CODE REVIEW

**Review by mcalmer — APPROVED** [review]

File: package/zypp-plugin-spacewalk.changes [hunk 1]
```diff
@@ -1,3 +1,9 @@
+-------------------------------------------------------------------
+Wed May 11 09:22:18 UTC 2022 - Cédric Bosdonnat <cbosdonnat@suse.com>
+
+- 1.0.13
+  * Log in before listing channels
```
Comment (mcalmer) [inline package/zypp-plugin-spacewalk.changes:5]: please add bugnumber(s)

Comment (cbosdo) [reply package/zypp-plugin-spacewalk.changes]: fixed

**Comment by cbosdo** [comment]
To sync with https://github.com/uyuni-project/uyuni/pull/5412

**Comment by cbosdo** [comment]
Tested the two PRs locally: the issue is gone and the few (8) failures in the CI were seemingly unrelated

**Review by mcalmer — APPROVED** [review]

**Comment by cbosdo** [comment]
@mcalmer can you merge? I tested with SUSE Manager 4.1 and it works

**Pull request merged**